Userland/Utilities/lsjails.cpp

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2022, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/JsonArray.h>
+#include <AK/JsonObject.h>
+#include <LibCore/File.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+
+ErrorOr<int> serenity_main(Main::Arguments)
+{
+    TRY(Core::System::pledge("stdio rpath"));
+    TRY(Core::System::unveil("/sys/kernel/jails", "r"));
+    TRY(Core::System::unveil(nullptr, nullptr));
+
+    auto jails_data = TRY(Core::File::open("/sys/kernel/jails", Core::OpenMode::ReadOnly));
```

Comment:
`Core::File` is deprecated for reading and writing, please use `Core::Stream::File` instead